pkgs/development/python-modules/eventlet/default.nix

```diff
@@ -59,6 +59,8 @@ buildPythonPackage rec {
     "test_patcher_existing_locks_locked"
     # broken with pyopenssl 22.0.0
     "test_sendall_timeout"
+    # broken on aarch64 (unclear if actual issue or not)
```

Comment:
Please make this conditional on aarch64.

Reply:
I would assume this test is flaky maybe? Doesn't seem like the failure is related to aarch64 specifically.

Reply:
If it is generally flake please update the config to reflect that.

Reply:
I think it's only broken on aarch64

Reply:
Test is now only disabled on aarch64.